uat/testing-features/pom.xml

```diff
@@ -114,6 +114,9 @@
                                 <resource>
                                     <directory>${basedir}/../../uat/custom-components/target/classes/recipes</directory>
                                 </resource>
+                                <resource>
+                                    <directory>${basedir}/../../uat/testing-features/target/classes/greengrass/recipes</directory>
```

Comment:
I am coping this recipes as well to the local store so that we can do a local install of the component when there is no network connectivity to ensure that when the network is back on it can still process the logs